docs/en/ingest-management/faq.asciidoc

```diff
@@ -152,13 +153,11 @@ you did previously with {beats} modules.
 [[does-agent-download-anything-from-internet]]
 == Does {agent} download anything from the Internet?
 
```

Comment:
@paul-tavares actually, can I get your comment on what you think we could say here?  I'd like our message now to be future compliant or we can just add a separate note now about Fleet Server usage.  I see from a recent PR that the artifacts like Trusted Apps and 'Exceptions' are actually sent from Kibana currently (which downloads them) and that they will be served via Fleet Server in the future, which will need to download them from somewhere (from Kibana or elsewhere?) before passing it on.

Reply:
Hi @EricDavisX - The artifacts will be stored in elasticsearch, so fleet-server will have access to them and does not have the need to download them. Currently, these artifacts are Endpoint specific and the agent (or fleet) has no awareness of them. That may change with the post you saw - as we move from kibana to fleet-server, these will have to also move, but I don't yet see a need for the agent to download anything. The downloading will be done by the Endpoint if installed.